static/app/views/performance/transactionSummary/transactionOverview/suspectSpans.tsx

```diff
@@ -0,0 +1,302 @@
+            id: example?.id ?? null,
+            spans: example?.spans,
+          };
+        });
+
+        return (
+          <Fragment>
+            <SuspectSpansHeader
+              location={location}
+              organization={organization}
+              projectId={projectId}
+              transactionName={transactionName}
+              pageLinks={pageLinks}
+            />
+            <GridEditable
+              isLoading={isLoading}
+              data={data}
+              columnOrder={SUSPECT_SPANS_TABLE_COLUMN_ORDER}
+              columnSortBy={[sortColumn]}
+              grid={{
+                renderHeadCell: renderHeadCellWithMeta(location, sortColumn),
+                renderBodyCell: renderBodyCellWithMeta({
+                  location,
+                  organization,
+                  transactionName,
+                }),
+              }}
+              location={location}
+            />
+          </Fragment>
+        );
+      }}
+    </SuspectSpansQuery>
+  );
+}
+
+const SUSPECT_SPANS_TABLE_COLUMN_TYPE: Omit<
+  Record<SuspectSpanTableColumnKeys, ColumnType>,
+  'spans'
+> = {
+  op: 'string',
+  description: 'string',
+  p75ExclusiveTime: 'duration',
+  count: 'integer',
+  sumExclusiveTime: 'duration',
+  id: 'string',
+};
+
+function renderHeadCellWithMeta(
+  location: Location,
+  sortColumn: GridColumnSortBy<SuspectSpanTableColumnKeys>
+) {
+  return (column: SuspectSpanTableColumn, _index: number): ReactNode => {
+    const columnType = SUSPECT_SPANS_TABLE_COLUMN_TYPE[column.key];
+    const align = fieldAlignment(column.key, columnType);
+
+    const canSort = columnType === 'integer' || columnType === 'duration';
+    const direction =
+      canSort && column.key === sortColumn.key ? sortColumn.order : undefined;
+
+    function generateSortLink() {
+      return {
+        ...location,
+        query: {
+          ...location.query,
+          spanSort: column.key,
+        },
+      };
+    }
+
+    return (
+      <SortLink
+        title={column.name}
+        align={align}
+        direction={direction}
+        canSort={canSort}
+        generateSortLink={generateSortLink}
+      />
+    );
+  };
+}
+
+function renderBodyCellWithMeta({
+  location,
+  organization,
+  transactionName,
+}: {
+  location: Location;
+  organization: Organization;
+  transactionName: string;
+}) {
+  return (
+    column: SuspectSpanTableColumn,
+    dataRow: SuspectSpanTableDataRow
+  ): ReactNode => {
+    const fieldRenderer = getFieldRenderer(column.key, SUSPECT_SPANS_TABLE_COLUMN_TYPE);
+    const rendered = fieldRenderer(dataRow, {location, organization});
+
+    if (column.key === 'id') {
+      if (defined(dataRow.id)) {
+        const worstSpan = dataRow.spans?.length
+          ? dataRow.spans.reduce((worst, span) =>
+              worst.exclusiveTime >= span.exclusiveTime ? worst : span
+            )
+          : null;
+
+        const hash = worstSpan ? `#span-${worstSpan.id}` : undefined;
```

Comment:
The anchor should probably not be hardcoded here, push this into a util function and have it use the same string.